private/buf/bufcli/bufcli.go

```diff
@@ -629,21 +630,30 @@ func newConnectClientConfigWithOptions(container appflag.Container, opts ...conn
 // up the token in the container or in netrc based on the address of each individual client.
 // It is then set in the header of all outgoing requests from clients created using this config.
 func NewConnectClientConfig(container appflag.Container) (*connectclient.Config, error) {
+	tokenSet, err := bufconnect.NewTokenSetProviderFromContainer(container)
```

Comment:
this should be renamed `envTokenProvider` once  `TokenFinder` is renamed to `TokenProvider` and `NewTokenProviderFromContainer` returns a new `TokenProvider`